settings/default_display_settings.json

```diff
@@ -0,0 +1,29 @@
+{
+  "blank_image": {
+    "type": "RGBA",
+    "size": "500, 300",
+    "color": "0, 0, 0, 0"
+  },
+  "name_label": {
+    "bg": "#1D1D1D",
+    "fg": "#FFFFFF",
+    "font": "Arial 18"
+  },
+  "display": {
+    "width": 500,
+    "height": 300,
+    "bg": "#1D1D1D"
+  },
+  "move_left_button": {
+    "text": "<",
+    "font": "Calibri 20",
+    "bg": "#1C1C1C",
+    "fg": "#FFFFFF"
+  },
+  "move_right_button": {
+    "text": ">",
+    "font": "Calibri 20",
+    "bg": "#1C1C1C",
```

Comment:
I don't think the background and foreground colours need to be in here due to them being covered in the other PR - they can stay hardcoded in the code for now as they were before

Reply:
Ok, I removed all back and foreground references and harcoded them back